minio/mc
# combine STS, SVC accounts into access keys with or without expiration

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
combine STS, SVC accounts into access keys with or without expiration

## Motivation and Context
from an end-user point of view, they are no different

## How to test this PR?
Configure LDAP and perform all the necessary activities to create accounts
with or without expiration. Then, observe the pretty printed output.

## Types of changes

## Checklist:
